### ISSUE TITLE
Please dont use $_SERVER

### ISSUE THREAD

**Issue by felloz** [vlucas/phpdotenv#556]
Is there a way to not override PHP's super global constant $_SERVER with the contents of the library?

**Comment by GrahamCampbell** [comment]
Yes. phpdotenv is very modular and flexible using composition. You can use the repository builder class without the default adapters, and then call create directly: https://github.com/vlucas/phpdotenv/blob/master/src/Dotenv.php#L83.
